### DISCUSSION TITLE
Any quick way to display plugin readme?

### DISCUSSION THREAD

**Discussion by InvisibleRasta** [ohmyzsh/ohmyzsh#10730] [Help]
Hello guys sometimes i forget what an alias does and i allways end up doing cat .oh-my-zsh/plugins/****  to check the readme.
is there any built-in functionality to display the readme?

**Comment by carlosala** [comment]
This doesn't exist by the moment, but maybe could be added to `omz` binary, it's a great idea!

**Comment by mcornella** [comment]
I think `omz plugin info <plugin>` is what you're looking for.

**Reply by carlosala** [reply]
> I think `omz plugin info <plugin>` is what you're looking for. 

I didn't knew that function🤯

**Reply by mcornella** [reply]
Yeah `omz` is not terribly well documented 😅

**Reply by carlosala** [reply]
We should fix that😉😉
